commons/connector/altoida/AltoidaPath.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "radarcns.connector.altoida",
+  "type": "record",
+  "name": "AltoidaPath",
```

Comment:
Since a single record only describes one location, this can be `AltoidaLocation` instead. In the description you could mention that this can be used to form a path.